src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -507,18 +508,20 @@
                     </Button>
                 </StackPanel>
             </Border>
-            <Border x:Name="TracePointer"
-                    Width="16"
-                    Height="18"
-                    Margin="48,50,0,0"
-                    HorizontalAlignment="Left"
-                    VerticalAlignment="Top"
-                    Visibility="Collapsed">
-                <FontIcon Foreground="Red"
-                          FontFamily="{StaticResource CalculatorFontFamily}"
-                          FontSize="18"
-                          Glyph="&#xE3B3;"/>
-            </Border>
+            <Grid x:Name="TracePointer"
```

Comment:
Minor bug, but if I have the TracePointer over a line, the trace value pop up disappears when my real mouse leaves the graph area. We can probably tackle it later if it isn't straightfoward, probably something to do with the state that is handled in grapher.cpp

Reply:
Yeah... I think this isn't straightforward.  The root cause of this is because we share a popup for the trace pointer and the mouse pointer.  We'd either have to write some complex logic on when to dismiss it, or disable the mouse trace cursor logic when active tracing is off.  Alternatively, we could have 2 popups and only show one at a time (with separate visual logic for each).